var/spack/repos/builtin/packages/root/package.py

```diff
@@ -34,14 +34,14 @@ class Root(CMakePackage):
     # Development version (when more recent than production).
 
     # Production version
-    version("6.30.02", sha256="51a09c86ffa94089abac76daa3adc20812efc6c93b427697b843b12d73e145de")
+    version("6.30.02", sha256="7965a456d1ad1ee0d5fe4769bf5a8fec291af684ed93db0f3080a9c362435183")
     version("6.30.00", sha256="0592c066954cfed42312957c9cb251654456064fe2d8dabdcb8826f1c0099d71")
-    version("6.28.10", sha256="fc6a2d6c7cba853b0cfd6bd9514c90e9df50e1743899bc1db0472ee6f4e65a0a")
+    version("6.28.10", sha256="69d6fdeb607e6b20bd02c757fa6217024c0b6132c1e9b1dff4d85d9a2bb7e51e")
     version("6.28.06", sha256="af3b673b9aca393a5c9ae1bf86eab2672aaf1841b658c5c6e7a30ab93c586533")
     version("6.28.04", sha256="70f7f86a0cd5e3f2a0befdc59942dd50140d990ab264e8e56c7f17f6bfe9c965")
     version("6.28.02", sha256="6643c07710e68972b00227c68b20b1016fec16f3fba5f44a571fa1ce5bb42faa")
     version("6.28.00", sha256="afa1c5c06d0915411cb9492e474ea9ab12b09961a358e7e559013ed63b5d8084")
-    version("6.26.12", sha256="229daa0749e3e31b6e0dedc58b6838dbfc1180b4aba4741883b617b0b4fbc966")
+    version("6.26.14", sha256="3b891ea23216234b90bf62397b036e9982607b687475f80d378e2f4bcb54a197")
```

Comment:
I am getting "81045a35a1958906c3eadecd9b01127a6087dadc19385af8e92ae5c84f06ce35" for this version (from 'spack checksum').

Reply:
This is the only version in the package that I cannot confirm.

Reply:
Yeah, not sure why I ended up with the wrong hash (how can hashes be so hard??). I'm now also getting:
```suggestion
    version("6.26.14", sha256="81045a35a1958906c3eadecd9b01127a6087dadc19385af8e92ae5c84f06ce35")
```